app-frontend/src/app/pages/user/organizations/organizations.js

```diff
@@ -1,6 +1,16 @@
 class UserOrganizationsController {
     constructor(organizations) {
-        this.organizations = organizations;
+        this.userRoles = organizations.roles;
```

Comment:
This structure sort of defeats the purpose of the resolve pattern. There are a few ways to approach this that would allow us to avoid this. The simplest is to add an additional property to the `pages.user` module's `resolve` (the example I'm going to use is teams):

```js
teamRoles: (userRoles) => {
    return _.uniqBy(userRoles, r => r.groupId)
        .filter(r => r.groupType === 'TEAM');
},
teams: ($q, teamRoles, teamService) => {
    const promises = teamRoles.map(r => teamService.getTeam(r.groupId));
    return $q.all(promises);
},
```

The cool thing about the way ui-router handles the resolves is that it intelligently handles both promises and non-promises. We should take advantage of that.

This would make `teamRoles` available in the constructor of any child controllers. The same could be done for organizations. 

The result would be that the constructor for this page controller would go from what you have here to:

```js
constructor(organizationRoles, organizations) {
    this.organizationRoles = organizationRoles;
    this.organizations = organizations;
}
```

which I feel is a bit more clear.

You used the same approach on the teams page and I'll leave a smaller comment there too.